excel-importer/src/main/java/uk/gov/hmcts/ccd/definition/store/excel/parser/AbstractDisplayGroupParser.java

```diff
@@ -141,9 +139,28 @@ public AbstractDisplayGroupParser(ParseContext parseContext, ShowConditionParser
 
         group.addDisplayGroupCaseFields(groupCaseFields);
         this.groupShowConditionColumn.ifPresent(column -> parseGroupShowCondition(column, group, groupDefinition.getValue()));
+        this.userRoleColumn.ifPresent(column -> parseUserRole(caseType, group, groupDefinition.getValue()));
         return ParseResult.Entry.createNew(group);
     }
 
+    private void parseUserRole(CaseTypeEntity caseType,
+                               final DisplayGroupEntity group,
+                               final  List<DefinitionDataItem> groupDefinition) {
+        if (groupDefinition.stream().filter(ddi -> ddi.getString(USER_ROLE) != null).count() > 1) {
```

Comment:
can we extract this block of code into a private method with a self explanatory name?

Reply:
Done

Reply:
done